docker/express/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node
+RUN apt-get update && apt-get install -y git
+RUN git clone https://github.com/MusicConnectionMachine/VisualizationG5.git
+WORKDIR VisualizationG5/express
+RUN npm install
+EXPOSE 3000
+CMD ["npm", "run", "dev"]
```

Comment:
If we want to use the container in development we should map the local volume so that changes during development are picked up by the container and the server is automatically restarted. We might use docker-compose for that and also to link the two containers. 

Otherwise I would start the server in production.